assets/javascript/companyNames.js

```diff
@@ -34,19 +34,9 @@ function getCompanyNames()
 	// We have the companies list, we gotta trim the list to 
 	// common stocks only. (type = "cs")
 	console.log(queryResponse[0].type);
-  for (i = 0; i < queryResponse.length; i++){
+  for (i = queryResponse.length-1; i > 0; i--){
 		var type = queryResponse[i].type.trim().toLowerCase();
-		var name = queryResponse[i].name.trim().toLowerCase();
```

Comment:
I don't think this row should be removed.

Reply:
It isn't being used in the function anymore. I had it there for debugging.

Reply:
I'm referring to 'var name'